recipes/algviz/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "algviz" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/algviz-{{ version }}.tar.gz
+  sha256: dc4c7b8e03f12d0ac71156af5e44a2807c7760230b7e71db6b9a1345313bd7ae
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<=3.10
+    - pip
+  run:
+    - python >=3.7
```

Metadata is says ` python_requires='>=3.7, <=3.10'`

```suggestion
    - python >=3.7,<=3.10
```